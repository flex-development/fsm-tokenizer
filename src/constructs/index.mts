/**
 * @file Entry Point - Constructs
 * @module vfile-tokenizer/constructs
 */

export { default as initialize } from '#constructs/initialize'
