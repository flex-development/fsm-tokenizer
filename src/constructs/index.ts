/**
 * @file Entry Point - Constructs
 * @module vfile-lexer/constructs
 */

export { default as eof } from './eof'
export { default as initialize } from './initialize'
