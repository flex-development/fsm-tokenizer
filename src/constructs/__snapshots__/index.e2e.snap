// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:constructs > should expose public api 1`] = `
[
  "initialize",
]
`;
